src/sentry/static/sentry/app/components/alerts.tsx

```diff
@@ -7,9 +7,15 @@ import AlertStore from 'app/stores/alertStore';
 import AlertMessage from 'app/components/alertMessage';
 import theme from 'app/utils/theme';
 
-const Alerts = createReactClass({
+type Alert = React.ComponentProps<typeof AlertMessage>['alert'];
+
+type State = {
+  alerts: Array<Alert>;
```

Comment:
We've generally been using `Alert[]`. I don't think this is a lintable thing though and is more of a nit.